libs/orders/src/lib/components/order-list/order-list.tsx

```diff
@@ -1,6 +1,6 @@
 import { memo, forwardRef, useMemo, type ForwardedRef } from 'react';
 import {
-  MAXGOINT64,
+  HALFMAXGOINT64,
```

Comment:
**thought (non-blocking)**: I understand *what* this is doing but not _why_. I imagine this is related to a core bug. I wonder if it is worth a comment either here or near line 154 just to explain why this is the limit, for future reference. 

At least git blame will bring someone to this PR and the bug it closes, so you can choose to ignore this

Reply:
updated but I am trying to find the core ticket